web/impact/impact/v1/views/office_hours_calendar_view.py

```diff
@@ -113,13 +119,14 @@ def _check_request_user_type(self, request):
         return True
 
     def _get_date_range(self, request):
-        date_spec = request.query_params.get("date_spec", None)
+        focal_date = request.query_params.get("focal_date", None)
+        calendar_span = request.query_params.get("calendar_span", "weekly")
```

Comment:
I would prefer to use "week" and "month" rather than "weekly" and "monthly" 
Reasoning: "weekly" and "monthly" indicate the period on which an event repeats: this is a weekly event, that one is monthly. Here, we want to know what span of time the user is interested in, either a week or a month.